CMakeLists.txt

```diff
@@ -46,20 +48,17 @@ if(VULKAN_FOUND)
                     "$ENV{VULKAN_SDK}/bin"
                     )
     endif()
-    if(DEFINED Vulkan_dxc_EXECUTABLE)
-        message("Found DirectX Shader Compiler under ${Vulkan_dxc_EXECUTABLE}")
-    elseif()
-        message("Couldn't find DirectX Shader Compiler executable, make sure it is present in Vulkan SDK or add it manually via Vulkan_dxc_EXECUTABLE env variable")
+    if(Vulkan_dxc_EXECUTABLE)
```

Comment:
It's probably better to use Vulkan_dxc_exe_FOUND for the boolean check

Reply:
Thanks for reviewing the PR.  Yes, I can change `Vulkan_dxc_EXECUTABLE` to `Vulkan_dxc_exe_FOUND`.  The semantics are the same, but your suggestion reads better.

Reply:
A quick update:  Can't use `Vulkan_dxc_exe_FOUND` in all places.  This variable is only returned by the **FindVulkan.cmake** module used by `find_package(Vulkan)`.  If you need to fall back to `find_program()` this variable will not be set.  So the code will need to be:

```
# search for Vulkan SDK
find_package(Vulkan)

if(Vulkan_FOUND)
    if(NOT Vulkan_dxc_exe_FOUND)
        find_program(Vulkan_dxc_EXECUTABLE
                    NAMES dxc
                    HINTS
                    "$ENV{VULKAN_SDK}/Bin"
                    "$ENV{VULKAN_SDK}/bin"
                    )
    endif()
    if(Vulkan_dxc_EXECUTABLE)
        message(STATUS "Found DirectX Shader Compiler under ${Vulkan_dxc_EXECUTABLE}")
    else()
        message(STATUS "Couldn't find DirectX Shader Compiler executable, make sure it is present in Vulkan SDK or add it manually via Vulkan_dxc_EXECUTABLE cmake variable. HLSL shaders won't be compiled.")
    endif()
endif()
```

The second test can't use `Vulkan_dxc_exe_FOUND`, but must use `Vulkan_dxc_EXECUTABLE`.  Same goes for the change in **sample_helper.cmake** below.

Reply:
Fixed in https://github.com/KhronosGroup/Vulkan-Samples/pull/1084/commits/1e3e85b9ab95a2e04b2d0180452ce9b68f509bff (`Vulkan_dxc_exe_FOUND` used in first test only, and `unset()` removed).